src/Control/SAMLController.php

```diff
@@ -147,6 +147,11 @@ public function acs()
             $guid = $auth->getNameId();
         }
 
+        $guidTransformation = Config::inst()->get(SAMLConfiguration::class, 'guid_transformation_callable');
+        if ($guidTransformation !== null) {
+            $guid = $guidTransformation($guid);
```

Comment:
Nice idea. How would you feel about adding an extension point instead? E.g. `$this->updateGuid($guid)` then extensions can call `strtoupper()` if they want to.

Reply:
was thinking about that but this means you have to actually create extension and do something in code
 typical use-case for this is simple upper/lower case so I thought that simple callable as string is much more simpler for that

Reply:
That's true. The difference is where you place the burden of maintenance - either on user code or in this module. Generally we place that burden on project or module developers which keeps our support footprint lighter.

Reply:
ok, if extension is prefered way how to do this...

added a fixup commit; will rebase if new commit will be ok, also I will change title and descriptions (will remove "callable")